openedx_proversity_reports/utils.py

```diff
@@ -198,7 +198,7 @@ def recurse_mark_complete(course_block_completions, latest_completion, block):
 
     if course_block_completions.get(block_key):
         block['complete'] = True
-        if block_key == latest_completion.full_block_key:
+        if block_key == latest_completion.block_key:
```

Comment:
Just asking, why do you need this change? @andrey-canon

Reply:
full_block_key is not supported on ginkgo version